packages/orbit-components/src/Alert/Alert.stories.tsx

```diff
@@ -210,16 +186,17 @@ export const OnlyTitle: Story = {
 
```

Comment:
this `OnlyTitle` story is the same as a `Default`. What do you think about deleting it?

Reply:
Definitely makes sense. Removed.